### PR TITLE
Update Speakers component for mobile

### PR DESCRIPTION
**Pull request by joeytepp**
## Proposed Change

Fixes: #127 #115

Make changes to the `Speakers` component so that it matches the mocks for mobile!

### How did you do this?
Found a great [npm package](https://github.com/akiran/react-slick) that lets you make swipeable carousels. From there added a new `SpeakerSlider` component and `SpeakerSliderDots` component (there is a way to include dots in the slider but they are only possible on the bottom which doesn't match the mocks) 

### Why are you choosing this approach?
To keep things modular and match the mocks

### Any open questions you want to ask code reviewers?
No

### List of changes:

  - Add new `SpeakerSlider` and `SpeakerSliderDots` components
  - Update `Speakers` component to implement new components
  - Add and update tests

## Pull Request Checklist:

  - [x] My submission passes all tests.
  - [x] I have lint my code locally prior to submission.
  - [x] I have written new tests for my core changes (where applicable).
  - [x] I have referenced all useful information (issues, tasks, etc) in the pull request.

### CODE REVIEW

**Comment by RobertWSaunders** [comment]
Deploy preview for *qhacks-website* ready!

Built with commit 442110a41c928e46d5c7efbca7696ebbe49fec07

https://deploy-preview-130--qhacks-website.netlify.com

File: src/components/SpeakerCard.js [hunk 1]
```diff
@@ -17,8 +17,11 @@ class SpeakerCard extends Component {
           css={{
```
Comment (RobertWSaunders) [inline src/components/SpeakerCard.js]: A few things with regards to CSS for the `SpeakerCard` and `SpeakerCardTextToggle`:

- The background color is not white like in the mocks.
- The names of the speakers are not all uppercase like in the mocks.
- The job titles are not the same font weight as in the mocks.
- The line height between the name and the job title are not the same as the mocks.
- The "LEARN MORE >>" does not hover to QHacks blue.

Can we update those things in this change as well please?

File: src/components/SpeakerSlider.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  render() {
+    const settings = {
+      dots: false,
+      arrows: false,
+      infinite: false,
+      onSwipe: (direction) => {
+        this.handleSwipe(direction);
+      }
+    };
```
Comment (RobertWSaunders) [inline src/components/SpeakerSlider.js]: Newline here.

**Comment by joeytepp** [comment]
@RobertWSaunders nice catches, turns out there was a lot of stuff that didn't match the mocks, made a few small tweaks as well to improve the speakers section, updated!

**Pull request merged**